### PR TITLE
Fix flaky testTransactionMetadataCleanup

### PR DESCRIPTION
**Pull request by viczhang861**
Attempt to fix https://github.com/prestodb/presto/issues/14835

Test is set to run 1000 times,  no failure observed in logging https://api.travis-ci.org/v3/job/712638157/log.txt  (terminated due to exceeding maximal log size)

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-tests/src/test/java/com/facebook/presto/tests/TestTransactionManagerIntegration.java [hunk 1]
```diff
@@ -13,31 +13,63 @@
...
 public class TestTransactionManagerIntegration
 {
+    private static final Logger log = Logger.get(TestTransactionManagerIntegration.class);
+
     @Test
     public void testTransactionMetadataCleanup()
             throws Exception
     {
         try (DistributedQueryRunner queryRunner = TpchQueryRunnerBuilder.builder().build()) {
+            log.info("Start running testTransactionMetadataCleanup");
```
Comment (rschlussel) [inline presto-tests/src/test/java/com/facebook/presto/tests/TestTransactionManagerIntegration.java:36]: do we still need this logging?

Comment (viczhang861) [reply presto-tests/src/test/java/com/facebook/presto/tests/TestTransactionManagerIntegration.java:36]: I am not sure whether this PR will completely solve the issue,  so additional logging of remaining transaction ids is another purpose of this PR.

Comment (rschlussel) [reply presto-tests/src/test/java/com/facebook/presto/tests/TestTransactionManagerIntegration.java:36]: that's covered in the assertTrue at line 41.  this logging just shows the beginning and end of the test.

Comment (viczhang861) [reply presto-tests/src/test/java/com/facebook/presto/tests/TestTransactionManagerIntegration.java:36]: Right,  assertTrue will show up in error message, then I need to search previous logging to match which queries are not cleaned. An example of 5 actual transaction ids are shown here. https://github.com/prestodb/presto/issues/14835

Comment (rschlussel) [reply presto-tests/src/test/java/com/facebook/presto/tests/TestTransactionManagerIntegration.java:36]: Got it. Can you create an issue to clean up the logging in e.g. 2 months if the error doesn't recur?

**Pull request merged**